lib/helpers/notifications.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:typed_data';
+import 'package:flutter/material.dart';
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+import '../main.dart';
+
+class LocalNotifications {
+  // Schedules a notification that specifies a different icon, sound and vibration pattern
+  Future<void> scheduleNotification(minute,sessionName) async {
+    var scheduledNotificationDateTime =
+    DateTime.now().add(Duration(minutes: minute));
```

Comment:
Remove this and use `sessionStartTime` directly.

Reply:
Done